src/User.php

```diff
@@ -1943,6 +1925,36 @@ public function getFromLDAP($ldap_connection, array $ldap_method, $userdn, $logi
                 $this->fields = Plugin::doHookFunction(Hooks::RETRIEVE_MORE_DATA_FROM_LDAP, $this->fields);
                 unset($this->fields['_ldap_result']);
             }
+
+            foreach ($import_fields as $k => $val) {
```

Comment:
Maybe this should be done before `$this->fields = Plugin::doHookFunction(Hooks::RETRIEVE_MORE_DATA_FROM_LDAP, $this->fields);`, in order to prevent any BC-break.